TestON/tests/PingallExample/ProcessOnos.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import multiprocessing
+import time
+
+class ProcessOnos(multiprocessing.Process):
+    def __init__(self,target = None, processID=None, name="", args=None):
+        super(ProcessOnos, self).__init__()
+        self.processID = processID
+        self.name = name
+        self.target = target
+        self.args = args
+        self.result = None
```

Comment:
Does this work? I wouldn't mind adding this to the core if it does. that could give us more options for parallelization